### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by oterral**
[test](https://mf-geoadmin3.int.bgdi.ch/teo_tests/be8ee5f/1472566631/index.html)

### CODE REVIEW

File: scripts/ol3cesium-geoadmin.json [hunk 1]
```diff
@@ -53,6 +53,8 @@
     "ol.format.GeoJSON#*",
     "ol.format.KML",
     "ol.format.KML#*",
+    "ol.format.MVT",
+    "ol.format.MVT#*",
```
Comment (loicgasser) [inline scripts/ol3cesium-geoadmin.json]: Probably not necessary just yet. :D

Comment (oterral) [inline scripts/ol3cesium-geoadmin.json]: yes indeed :)

**Comment by gjn** [comment]
This needs a rebase.

**Comment by oterral** [comment]
rebased

File: test/karma-conf.mako.js [hunk 2]
```diff
@@ -12,11 +12,13 @@ module.exports = function(config) {
   // list of files / patterns to load in the browser
   files: [
   % if mode == 'release':
+    'style/app.css',
```
Comment (gjn) [inline test/karma-conf.mako.js]: We do we need css now in unit tests?

File: test/specs/NetworkStatusService.spec.js [hunk 3]
```diff
@@ -0,0 +1,298 @@
+describe('ga_networkstatus_service', function() {
+
+  describe('httpInterceptor', function() {
+    var httpI, gaDebounce, gaNetowrkStatus, $q, gaWaitCursor, $timeout;
```
Comment (gjn) [inline test/specs/NetworkStatusService.spec.js]: Type: gaNeetowrkStatus

File: test/karma-conf.mako.js (hunk 2 above)
Comment (oterral) [inline test/karma-conf.mako.js]: Yes because gaPrint service copy the `<link href="/lib/app.css"....>`  from the initial page.

**Pull request merged**